main.tf

```diff
@@ -1,4 +1,4 @@
 resource "aws_instance" "test" {
   ami           = "ami-08569b978cc4dfa10"
-  instance_type = "t2.small"
+  instance_type = "t9.large"
```

Comment:
🚫 **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>"t9.large" is an invalid value as instance_type